### DISCUSSION TITLE
DS3502

### DISCUSSION THREAD

**Discussion by ThornySpore** [micropython/micropython#10223] [Hardware & Peripherals] [answered]
Hello!
I'm trying to work with adafruit's ds3502 digital potentiometer breakout. I ran i2c.scan(), which returned the expected address. The issue comes when I try to write to the wipers register, it just will not work no matter what I try, when I read the register it shows `b'/x'` and voltage between RW and RL stays the same. I hope someone here has an answer. For testing purposes I ran their example code for circuitpython and it worked well, but I cannot use CP for the rest of my project and I'm just not experienced enough to figure out the difference between how their ds3502 library works vs using the write to mem function. Any help is appreciated! 
![PXL_20221212_162718699](https://user-images.githubusercontent.com/120606098/207731054-341b7bb4-682b-40e3-ac3a-8afc8e3e8c95.jpg)
![PXL_20221212_162620534 MP](https://user-images.githubusercontent.com/120606098/207731078-7397ceb5-00d1-4e10-89dd-b0be505f76b3.jpg)

**Comment by dlech** [comment, marked as answer]
Should you be using a backslash (`\`) instead of a forward slash (`/`) in your bytes? `b'\x40'` is a single byte with the value 0x40 while `b'/x40`' is 4 bytes with the values 0x2f, 0x78, 0x34, 0x30.

**Reply by ThornySpore** [reply]
I'll give that a try first thing in the morning and be sure to let you know how that works. Thank you for your quick response!

**Reply by ThornySpore** [reply]
Worked as expected 🙂
